tasks/update-cnb-dependency/main_test.go

```diff
@@ -353,6 +366,10 @@ func testUpdateCNBDependencyTask(t *testing.T, when spec.G, it spec.S) {
 			var buildpackTOML BuildpackTOML
 			_, err = toml.DecodeFile(filepath.Join(outputDir, "buildpack.toml"), &buildpackTOML)
 			require.NoError(t, err)
+
```

Comment:
nit: Not equal spacing in all the additions - either all of that line (`var deps Dependencies`) should have a line above them, or none of them